### PR TITLE
New permission-maintainers

### PR DESCRIPTION
**Pull request by timbonicus**
Adds new members of the @backstage/permission-maintainers team: @sennyeya and @lynettelopez

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: OWNERS.md [hunk 1]
```diff
@@ -73,16 +73,18 @@ Team: @backstage/permission-maintainers
...
+| Name                 | Organization | Team            | GitHub                                          | Discord          |
+| -------------------- | ------------ | --------------- | ----------------------------------------------- | ---------------- |
+| Ainhoa Larumbe       | Spotify      | Imaginary Goats | [ainhoaL](http://github.com/ainhoaL)            | ainhoa#8085      |
+| Aramis Sennyey       | Spotify      | Imaginary Goats | [sennyeya](https://github.com/sennyeya)         | Aramis#7984      |
+| Claire Casey         | Spotify      | Imaginary Goats | [clairelcasey](http://github.com/clairelcasey)  | clairecasey#2710 |
+| Eric Peterson        | Spotify      | Imaginary Goats | [iamEAP](http://github.com/iamEAP)              | iamEAP#3058      |
+| Harry Hogg           | Spotify      | Imaginary Goats | [HHogg](http://github.com/HHogg)                | simplex#3451     |
+| Joon Park            | Spotify      | Imaginary Goats | [Joonpark13](http://github.com/Joonpark13)      | Sixpool#5060     |
+| Lynette Lopez        | Spotify      | Imaginary Goats | [lynettelopez](https://github.com/lynettelopez) | -                |
```
Comment (lynettelopez) [inline OWNERS.md:84]: Not super important, but you can put my Discord handle down as `lynettelopez` too!

Comment (Joonpark13) [reply OWNERS.md:84]: Do you know the 4 digit number after your discord username?

**Review by Joonpark13 — APPROVED** [review]
I think this is the first time I ran into a change that is better viewed in unified diff instead of split diff (🌶️ hot take, I know...)

**Comment by freben** [comment]
@Joonpark13 especially with the Hide whitespace option!

File: OWNERS.md (hunk 1 above)
Comment (freben) [reply OWNERS.md:84]: After their [name system revamp](https://support.discord.com/hc/en-us/articles/12620128861463-New-Usernames-Display-Names) this is actually OK!

**Review by freben — APPROVED** [review]
Pre-approving, with the caveat that @lynettelopez wants to be added too before merge

File: OWNERS.md (hunk 1 above)
Comment (lynettelopez) [reply OWNERS.md:84]: Yup, this is my new username after their revamp!

**Pull request merged**